### PR TITLE
Use enum VideoQuality/TrackType without exporting PBType.

### PR DESCRIPTION
**Pull request by cloudwebrtc**

### CODE REVIEW

File: lib/src/publication/remote.dart [hunk 1]
```diff
@@ -235,11 +235,11 @@ class RemoteTrackPublication<T extends RemoteTrack>
   }
 
   @Deprecated('use setVideoQuality() instead')
```
Comment (lukasIO) [inline lib/src/publication/remote.dart:237]: if this is deprecated and the PR is part of v2, we can probably remove it here (unless this is removed in the deprecation PR already, then ignore me :) )

**Review by lukasIO — APPROVED** [review]
lgtm!

File: lib/src/publication/remote.dart (hunk 1 above)
Comment (cloudwebrtc) [reply lib/src/publication/remote.dart:237]: yes, he will be deleted in another PR. I changed it here just to not break the dart analyzer.

**Review by davidzhao — APPROVED** [review]
nice work!

one question: is it necessary to change it to lower case enums? I think we still use some upper case enums in the code base? Question is if it's worth making users change all of these enums.

**Comment by cloudwebrtc** [comment]
> nice work!
> 
> one question: is it necessary to change it to lower case enums? I think we still use some upper case enums in the code base? Question is if it's worth making users change all of these enums.

Makes sense, but changing to uppercase will cause a dart-style prompt requiring `lowerCamelCase`, unless we change `analysis_options.yaml` to disable lowerCamelCase mandatory requirements.

image

**Comment by davidzhao** [comment]
> Makes sense, but changing to uppercase will cause a dart-style prompt requiring `lowerCamelCase`, unless we change `analysis_options.yaml` to disable lowerCamelCase mandatory requirements.

Got it. I think that would be preferable instead of forcing users to update all the enum references. wdyt?

**Pull request merged**